index.js

```diff
@@ -224,4 +224,43 @@ window.onload = () => {
   contents.forEach((content) => {
     observer.observe(content);
   });
+
+  //   document.querySelector("input").onchange = () => {
+  //     alert("hi")
+  //   }
+
+  const email = document.querySelector('.email');
+  const emailError = document.querySelector('.email-error');
+  const error = 'Email has to be in lower case';
+
+  const shaking = (element) => {
+    element.classList.remove('shaking');
+    // void element.offsetWidth;
```

Comment:
- [ ] Please kindly remove this commented lines of code to make your work clean. 👍